data/deduplicator/hash_deactivate_old_test.go

```diff
@@ -100,6 +117,23 @@ var _ = Describe("HashDeactivateOld", func() {
 				testDataset.DeviceManufacturers = app.StringArrayAsPointer([]string{"Ant", "Zebra", "Medtronic", "Cobra"})
 				Expect(testFactory.CanDeduplicateDataset(testDataset)).To(BeTrue())
 			})
+
+			DescribeTable("returns true when",
+				func(deviceManufacturer string, deviceModel string) {
+					testDataset.DeviceManufacturers = app.StringArrayAsPointer([]string{deviceManufacturer})
+					testDataset.DeviceModel = app.StringAsPointer(deviceModel)
+					Expect(testFactory.CanDeduplicateDataset(testDataset)).To(BeTrue())
+				},
+				Entry("is Medtronic 523", "Medtronic", "523"),
```

Comment:
as it is is a case sensitive match maybe we should so that with a test?

Reply:
I try to stay away from negative testing as it is an endless process. Should we test that it fails without spaces? You can see how that would get overwhelming. The main point here was to ensure that all of the valid device manufacturer/model combinations do, in fact, work.